packages/o11ylogsdb/test/engine-extended.test.ts

```
@@ -0,0 +1,154 @@
+  it("unflushed records are not in stats", () => {
+    const store = new LogStore({ rowsPerChunk: 16 });
+    for (let i = 0; i < 5; i++) store.append(resource, scope, rec(i));
+    // No flush — records are in-flight
+    expect(store.stats().chunks).toBe(0);
+    expect(store.stats().totalLogs).toBe(0);
+  });
+
+  it("chunksClosed counter increments correctly across auto-freeze and manual flush", () => {
+    const store = new LogStore({ rowsPerChunk: 4 });
+    // First 4 records auto-freeze a chunk
+    let lastStats;
```

Biome (2.4.13)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)